app/controllers/user.js

```diff
@@ -0,0 +1,8 @@
+const { newUser } = require('../services/users');
+
+exports.newUser = (req, res, next) => {
+  const { firstName, lastName, email, password } = req.body;
```

Comment:
I think you can restrict the fields by applying destructuring in the service and keep the controller as clear as you can do it, some like this: 
```
exports.myController = (req, res, next) =>  
    myService(req.body)
      .then(...)
      .catch(...)
```

what do you think?